### DISCUSSION TITLE
How can I get username instead of user id?

### DISCUSSION THREAD

**Discussion by serkan-bayram** [pocketbase/pocketbase#3726] [Q&A] [answered]
This is my table 

![image](https://github.com/pocketbase/pocketbase/assets/68502470/32931412-0d5c-4e1e-a6ce-bfdef74c5f66)

This is the code

```js
    records = await pb.collection("ratings").getFullList({
      sort: "-updated",
      filter: `menu="${menuId}"`,
    });

    console.log(records);
```

This is an example output

![image](https://github.com/pocketbase/pocketbase/assets/68502470/2a9bf366-591c-4485-9592-c8a2f9c1c11e)

I need to have "username" instead of "user", can I do this or do I need to make an another query?

**Comment by ganigeorgiev** [comment, marked as answer]
Relations are associated only via id. It is not possible at the moment to replace the "user" id value in the above  with the username.

If you want to access the user's username, you can use `expand` (requires to have View API rule access to the related collection):

```js
records = await pb.collection("ratings").getFullList({
    sort: "-updated",
    filter: `menu="${menuId}"`,
    expand: "user"
});
```
(_for more details about `expand` you could check https://pocketbase.io/docs/working-with-relations/_)
